spara.js

```diff
@@ -0,0 +1,62 @@
+// get a list of songs. The endpoint can be named anything,
```

Comment:
If you want to keep the explanation file in the project, I would suggest organising it a little to be easier to outsiders to understand.